web/src/scenes/sign-in-with-world-id/Redirects/index.tsx

```diff
@@ -38,11 +40,22 @@ export const Redirects = memo(function Redirects() {
             onDelete={() => deleteRedirect({ id: redirect.id })}
           />
         ))}
+        {addRedirectFormShown && (
+          <RedirectInput
+            value=""
+            onChange={(value) => {
+              addRedirect({ uri: value }).then(() => {
+                setAddRedirectFormShown(false);
```

Comment:
@michaelfurutin This causes a little blink when you add a new redirect.

Reply:
Resolved with implementing optimistic update and hidding form before success response.